client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -7,17 +7,27 @@ import React from 'react';
 /**
  * Internal Dependencies
  */
+import wpcom from 'lib/wp';
```

Comment:
This was a special departure I discussed w/ @rralian since I didn't make sense to complicate the update w/ all the redux overhead.